drivers/counter/counter_nrfx_rtc.c

```diff
@@ -13,7 +13,8 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL);
 
-#define RTC_CLOCK 32768
+#define LF_FREQ 32768
```

Comment:
Is there as system-level define for that? If no, could we add clock-frequency node to the RTC?

Reply:
as already mentioned in another comment, i will handle DTS related stuff in separate commit. ok?

Reply:
as #16135 went in, things have been moved to device tree.